jbmc/regression/jbmc/NondetEnumOpaqueReturn/indirect/IndirectNestedEnumOnReturn.java

```diff
@@ -0,0 +1,8 @@
+package indirect;
+
+public class IndirectNestedEnumOnReturn {
+    public void main() throws ClassNotFoundException, IllegalAccessException, InstantiationException {
+        PrimaryInterface primary = (PrimaryInterface)Class.forName("PrimaryInstance").newInstance();
+        ClassWithEnumField enumClass = primary.getEnumOwningClass();
```

Comment:
:no_entry_sign: I can't reproduce the invariant violation on develop with this test. There should be an additional line that accesses the enum, e.g.
```
        enumClass.enumInstance.isActive();
```
(where `enumInstance` would have to be public).

Reply:
It's not really to do with the enum being called - it's specifically that a class instance with an enum field is a return value and not recognised as required anywhere else and then being nondet initialised. But it certainly isn't throwing the violation as it should, so I'll fix it up.